.npmignore

```diff
@@ -0,0 +1,5 @@
+/test
```

Comment:
instead `.npmignore` I would keep the `files` section inside `package.json` to list what ned to be published to NPM